docs/changelog.rst

```diff
@@ -15,8 +17,8 @@ Changelog
 * :feature:`3470` Users can now customize the date input format.
 * :feature:`2918` Users can now select how the percentage column is calculated in the dashboard. Either based on total net value or based on current group (assets, liabilities, or nft).
 * :feature:`3753` Application users will now be able to (copy, cut and paste) using the context menu .
-* :feature:`3525` Users can now see the NFT icon at the NFT Balance table, and Non Fungible Balance table.
-* :feature:`3176` Amount inputs will now adhere to the user specified amount formatting.
+* :feature:`3818` Users can now see the NFT icon at the NFT Balance table, and Non Fungible Balance table.
+* :feature:`3814` Amount inputs will now adhere to the user specified amount formatting.
```

Comment:
3818 and 3814 are the PR numbers, not the issue numbers, no need to change these
3525 and 3176 are the issues for these.